packages/expo-camera/build/Camera.d.ts

```diff
@@ -22,8 +22,8 @@ export default class Camera extends React.Component<CameraProps> {
     static Constants: ConstantsType;
     static ConversionTables: {
         type: Record<"front" | "back", string | number | undefined>;
-        flashMode: Record<"on" | "off" | "auto" | "torch", string | number | undefined>;
-        autoFocus: Record<"on" | "off" | "auto" | "singleShot", string | number | boolean | undefined>;
+        flashMode: Record<"auto" | "off" | "on" | "torch", string | number | undefined>;
+        autoFocus: Record<"auto" | "off" | "on" | "singleShot", string | number | boolean | undefined>;
```

Comment:
@Simek, the following changes are auto generated, do you think they can cause issue for rebuilding the package files?

I reverted the doc change for Expo Camera and I can see these lines in the file are still happening.